dataflux_pytorch/lightning/dataflux_lightning_checkpoint.py

```diff
@@ -52,18 +51,17 @@ def _parse_gcs_path(self, path: Union[str, Path]) -> str:
             bucket_name, prefix = split
         if not bucket_name:
             raise ValueError("Bucket name must be non-empty")
-        if not self.bucket:
-            self.bucket = self.storage_client.bucket(bucket_name)
-        return prefix
+        return prefix, bucket_name
```

Comment:
nit: It might be more intuitive to return these in the reverse order, since the bucket name comes before the prefix in the original string

Reply:
Agreed. Changed the order.